importer_client/python/tools/timesketch_importer.py

```diff
@@ -462,6 +482,34 @@ def main(args=None):
         help=("Path to the file that is to be imported."),
     )
 
+    config_group.add_argument(
+        "--analyzer_names",
+        "--analyzer-names",
+        type=comma_separated_list,
+        action="store",
+        dest="analyzer_names",
+        default=[],
+        help=(
+            "Set of analyzers that we will automatically run right after the "
+            "timelines are uploaded. The input needs to be the analyzers names."
+            "Provided as a comma-separated string."
+        ),
+    )
```

Comment:
Instead of using a comma separated string (potentially error prone and requires extra parsing) I would recommend going with the [argparse nargs](https://docs.python.org/3/library/argparse.html#nargs) functionality to handle multiple analyzers.